controllers/imagehub/api/v1/image_types.go

```diff
@@ -113,9 +115,10 @@ type ImageDetailInfo struct {
 	// URL sealos cloud ui endpoint
 	URL string `json:"url,omitempty"`
 	// ID Arch Size Ctime should use buildah inspect to get infor.
-	ID    string      `json:"ID,omitempty"`
-	Arch  string      `json:"arch,omitempty"`
-	Size  int64       `json:"size,omitempty"`
+	ID   string `json:"ID,omitempty"`
+	Arch string `json:"arch,omitempty"`
+	Size int64  `json:"size,omitempty"`
+	//+kubebuilder:default:="1970-01-01T00:00:00Z"
 	CTime metav1.Time `json:"creatTime"`
```

Comment:
`CreateTime` is better than `CTime`.

Reply:
Or `ImageBuildTime`?

Reply:
use `CreateTime`, since json is `creatTime` and not easy to change.